SofaKernel/framework/sofa/core/behavior/LinearSolver.h

```diff
@@ -141,12 +141,28 @@ class SOFA_CORE_API LinearSolver : public BaseLinearSolver
     /// @param result the variable where the result will be added
     /// @param fact integrator parameter
     /// @return false if the solver does not support this operation, of it the system matrix is not invertible
-    virtual bool buildComplianceMatrix(const sofa::core::ConstraintParams* cparams, defaulttype::BaseMatrix* result, SReal fact)
+    virtual bool computeComplianceMatrix(const sofa::core::ConstraintParams* cparams, defaulttype::BaseMatrix* result, SReal fact)
     {
         SOFA_UNUSED(cparams);
         SOFA_UNUSED(result);
         SOFA_UNUSED(fact);
-        serr << "Error buildComplianceMatrix has not been implemented" << sendl;
+        msg_error() << "Error computeComplianceMatrix has not been implemented" << sendl;
```

Comment:
Minor comment, there is no need to write "Error"...because msg_error already add that to the string.